modules/log6x-6-1-0-rn.adoc

```diff
@@ -0,0 +1,53 @@
+= Logging 6.1.0 Release Notes
+// Module included in the following assemblies:
+//
+
+:mod-docs-content-type: REFERENCE
+[id="logging-release-notes-6-1-0_{context}"]
+
+This release includes link:https://access.redhat.com/errata/RHBA-2024:9038[{OCP-short}{logging} Release 6.1.0].
+
+[id="openshift-logging-release-notes-6-1-0-enhancements"]
+== New Features and Enhancements
+
+=== Log Collection
+
+* This enhancement adds the source `iostream` to the attributes sent from collected container logs. The value is set to either `stdout` or `stderr` based on how the collector received it. (link:https://issues.redhat.com/browse/LOG-5292[LOG-5292])
+
+* With this update, the default memory limit for the collector increases from 1024Mi to 2024Mi. Users should adjust resource limits based on their cluster’s specific needs and specifications. (link:https://issues.redhat.com/browse/LOG-6072[LOG-6072])
+
+* With this update, users can now set the syslog output delivery mode to either `AtLeastOnce` or `AtMostOnce.` (link:https://issues.redhat.com/browse/LOG-6355[LOG-6355])
```

Comment:
Do we need to show users how to set this?

Reply:
Added where those fields are.